e2e/core/settings-component.e2e.ts

```diff
@@ -0,0 +1,118 @@
+            expect(settingsPage.getBpmHostUrl()).not.toEqual('http://adfdev.envalfresco1.com', 'The Settings changes are saved');
+
+        });
+
+        it('[C291947] Should not save ECM Settings changes when User clicks Back button', () => {
+            settingsPage.setProvider(settingsPage.getEcmOption(), 'ECM');
+            settingsPage.setContentServicesURL('http://adfdev.envalfresco1.com');
+            settingsPage.clickBackButton();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).not.toEqual('ECM', 'The Settings changes are saved');
+            expect(settingsPage.getBpmHostUrl()).not.toEqual('http://adfdev.envalfresco1.com', 'The Settings changes are saved');
+
+        });
+
+        it('[C291948] Should save ALL Settings changes when User clicks Apply button', () => {
+            settingsPage.setProviderEcmBpm();
+            loginPage.waitForElements();
+            settingsPage.goToSettingsPage();
+            expect(settingsPage.getSelectedOptionText()).toEqual('ALL', 'The Settings changes are not saved');
+            expect(settingsPage.getBpmHostUrl()).toEqual('http://adfdev.envalfresco.com', 'The BPM Settings changes are not saved');
+            expect(settingsPage.getEcmHostUrl()).toEqual('http://adfdev.envalfresco.com', 'The ECM Settings changes are not saved');
+
+        });
+
+        it('[C291949] Should have field validation for Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearContentServicesURL();
+            settingsPage.ecmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291950] Should have field validation for Process Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmAndBpmOption(), 'ALL');
+            settingsPage.clearProcessServicesURL();
+            settingsPage.bpmText.sendKeys(protractor.Key.TAB);
+            settingsPage.checkValidationMessageIsDisplayed();
+            settingsPage.checkApplyButtonIsDisabled();
+        });
+
+        it('[C291951] Should not be able to sign in with invalid Content Services Url', () => {
+            settingsPage.setProvider(settingsPage.getEcmOption(), 'ECM');
+            settingsPage.setContentServicesURL('http://localhost:7070');
+            settingsPage.clickApply();
+            loginPage.waitForElements();
+            loginPage.enterUsername(adminUserModel.id);
+            loginPage.enterPassword(adminUserModel.password);
+            loginPage.clickSignInButton();
+            expect(loginPage.getLoginError()).toMatch('Request has been terminated ' +
+                'Possible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unloaded, etc.');
```

Comment:
You can save this in a variable and use it in both places.

Reply:
moved it to a variable.